src/app/cluster/cluster-details/cluster-details.component.ts

```diff
@@ -106,6 +109,10 @@ export class ClusterDetailsComponent implements OnInit, OnDestroy {
       .getCurrentUserGroup(this.projectID)
       .subscribe(userGroup => (this._currentGroupConfig = this._userService.getCurrentUserGroupConfig(userGroup)));
 
+    if (this._guidedTourService.isTourInProgress()) {
```

Comment:
The biggest issue here is that directive usage should be independent and not require logic changes inside the components that use it. This `if` is reused 65 times across this change. We can't have such hard dependency. It should be minimized to some common service/interceptor etc. so that we won't have to remember about adding it manually when using it.